src/pages/projectSettings/tabs/GithubCommitQueueTab/GithubCommitQueueTab.tsx

```diff
@@ -100,7 +107,73 @@ export const GithubCommitQueueTab: React.VFC<TabProps> = ({
         schema={schema}
         uiSchema={uiSchema}
         disabled={isProduction() && !githubWebhooksEnabled} // TODO: Remove once EVG-16608 is fixed
+        validate={validateConflicts}
       />
     </>
   );
 };
+
+const validate = (projectType, repoData, versionControlEnabled) => (
+  formData,
+  errors
+) => {
+  const {
+    github: {
+      prTestingEnabled,
+      prTesting,
+      githubChecksEnabled,
+      githubChecks,
+      gitTagVersionsEnabled,
+      gitTags,
+    },
+    commitQueue: { enabled, patchDefinitions },
+  } = formData;
+
+  const getError = getVersionControlError(versionControlEnabled, projectType);
+
+  if (
+    getError(
+      prTestingEnabled,
+      prTesting?.githubPrAliasesOverride,
+      prTesting?.githubPrAliases,
+      repoData?.github?.prTesting?.githubPrAliases
+    ) === ErrorType.Error
+  ) {
+    errors.github.prTesting.addError("Missing Patch Definition");
+  }
+
+  if (
+    getError(
+      githubChecksEnabled,
+      githubChecks?.githubCheckAliasesOverride,
+      githubChecks?.githubCheckAliases,
+      repoData?.github?.githubChecks?.githubCheckAliases
+    ) === ErrorType.Error
+  ) {
+    errors.github.prTesting.addError("Missing Commit Check Definition");
+  }
+
+  if (
+    getError(
+      gitTagVersionsEnabled,
+      gitTags?.gitTagAliasesOverride,
+      gitTags?.gitTagAliases,
+      repoData?.github?.gitTags?.gitTagAliases
+    ) === ErrorType.Error
+  ) {
+    errors.github.prTesting.addError("Missing Git Tag Definition");
+  }
+
+  if (
+    getError(
+      enabled,
+      patchDefinitions?.commitQueueAliasesOverride,
+      patchDefinitions?.commitQueueAliases,
+      repoData?.commitQueue?.patchDefinitions?.commitQueueAliases
+    ) === ErrorType.Error
+  ) {
+    errors.github.prTesting.addError("Missing Commit Queue Patch Definition");
```

Comment:
Would it be possible for you to include a screenshot with an example of one of these errors being displayed?

Reply:
These errors are actually never displayed -- their primary use case is in disabling save when one of them is encountered. They represent the same errors in the screenshots in the description.

Ideally we would be able to combine the error messaging found in `getErrors.ts` with this function, but RJSF sadly does not natively support warnings (as opposed to errors) so they had to be split up :(